nuxeo-runtime/nuxeo-runtime-metrics/src/main/java/org/nuxeo/runtime/metrics/MetricsDescriptor.java

```diff
@@ -268,6 +268,113 @@ public void disable(MetricRegistry registry) {
 
     }
 
+    /**
+     * @since 10.3
+     */
+    @XObject(value = "statsDReporter")
+    public static class StatsDDescriptor {
+
+        public static final String ENABLED_PROPERTY = "metrics.statsd.enabled";
+
+        public static final String HOST_PROPERTY = "metrics.statsd.host";
+
+        public static final String PORT_PROPERTY = "metrics.statsd.port";
+
+        public static final String PERIOD_PROPERTY = "metrics.statsd.period";
+
+        public static final String PREFIX_PROPERTY = "metrics.statsd.prefix";
+
+        /**
+         * A list of metric prefixes that if defined should be kept reported, separated by commas
+         */
+        public static final String ALLOWED_METRICS_PROPERTY = "metrics.statsd.allowedMetrics";
+
+        /**
+         * A list of metric prefixes that if defined should not be reported, separated by commas
+         */
+        public static final String DENIED_METRICS_PROPERTY = "metrics.statsd.deniedMetrics";
+
+        public static final String DEFAULT_ALLOWED_METRICS = GraphiteDescriptor.DEFAULT_ALLOWED_METRICS;
+
+        public static final String DEFAULT_DENIED_METRICS = GraphiteDescriptor.DEFAULT_DENIED_METRICS;
+
+        public static final String ALL_METRICS = "ALL";
+
+        @XNode("@enabled")
+        protected Boolean enabled = Boolean.valueOf(Framework.getProperty(ENABLED_PROPERTY, "false"));
+
+        @XNode("@host")
+        public String host = Framework.getProperty(HOST_PROPERTY, "127.0.0.1");
+
+        @XNode("@port")
+        public Integer port = Integer.valueOf(Framework.getProperty(PORT_PROPERTY, "8125"));
+
+        @XNode("@periodInSecond")
+        public Integer period = Integer.valueOf(Framework.getProperty(PERIOD_PROPERTY, "10"));
```

Comment:
I see this use of framework properties is already in other descriptors, but note that it's not what we want. We dont want to use framework properties in code anymore. There should be a default contribution provided by a template with properties expanded from nuxeo.conf.

Reply:
For durations like `period` we now have `Duration` directly available in XMap (https://jira.nuxeo.com/browse/NXP-25402)

Reply:
I agree, but out of the scope for 1pt feature